mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -463,6 +472,54 @@ is set to true.`,
 							},
 							Description: `Configuration of Query Insights.`,
 						},
+						"password_validation_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"min_length": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(0, 2147483647),
+										AtLeastOneOf: passwordValidationPolicyKeys,
+										Description:  `Minimum number of characters allowed.`,
+									},
+									"complexity": {
+										Type:         schema.TypeString,
+										Optional:     true,
+										ValidateFunc: validation.StringInSlice([]string{"COMPLEXITY_DEFAULT", "COMPLEXITY_UNSPECIFIED"}, false),
+										AtLeastOneOf: passwordValidationPolicyKeys,
+										Description:  `Password complexity.`,
+									},
+									"reuse_interval": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(0, 2147483647),
+										AtLeastOneOf: passwordValidationPolicyKeys,
+										Description:  `Number of previous passwords that cannot be reused.`,
+									},
+									"disallow_username_substring": {
+										Type:         schema.TypeBool,
+										Optional:     true,
+										AtLeastOneOf: passwordValidationPolicyKeys,
+										Description:  `Disallow username as a part of the password.`,
+									},
+									"password_change_interval": {
+										Type:         schema.TypeString,
+										Optional:     true,
+										AtLeastOneOf: passwordValidationPolicyKeys,
+										Description:  `Minimum interval after which the password can be changed. This flag is only supported for PostgresSQL.`,
+									},
+									"enable_password_policy": {
```

Comment:
Tested it locally and it seems that we need to set `enable_password_policy` to `true` to active password policy, or to say, to config other fields under `password_validation_policy` object. Therefore, should we set `enable_password_policy` field to be required? What's your thought?

Reply:
That is correct, I have made the changes.